lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -283,3 +283,19 @@ def __init__(
         **kwargs,
     ):
         super().__init__(child, f=standardizer.standardize, args=path, kwargs=kwargs)
+
+class IgnoreErrors(standardizer)
+
+   """
+    A class for applying the behavior of a standardizer to log errors and continue when encountering null values.
+
+    This class allows for the execution of standardization logic not to fail when encountering null key:value pairs. It will
+    instead log a warning stating what key:value pairs in what documents were missing.
+    """
+    
+    def ignore_errors(doc, standardizer, key_path: list[str]):
+        try:
+            doc = standardizer.standardize(doc, key_path=key_path)
+        except:
```

Comment:
prob wanna `except KeyError` if you're only trying to catch those. If you're ok with any exception, then you prob want something like 
```python
except Exception as e:
    print(e)
```

Reply:
Ack